people/yumao/page.en.md

```diff
@@ -0,0 +1,152 @@
+
+During that time, Yumao got many mental supports. She described the care from Neko like this:
+
+> "Even those things were dark and painful, which had been hidden, you still observed and felt them. Maybe you understanded more then myself. How wonderful... You also found my scattered identity, then lifted me out of abyss. That was the first time I was been touched by others."
+
+However, they were forced to separate.
+After that, Neko was brought back to Yunnan by family, and was sent to correctional school.
+Then, it was reported that Neko had already died by suicide.
+Yumao was extremely grieved after hearing the news.
+She continuously sent messages about missing Neko in their Telegram channel.
+She quoted *Cassiopeia(カシオペヤ)* , which was a song they used to listen together, to express her grief.
+
+Although she also found that the suicide of Neko was complicated and confusing according to some routes.
+But there was no more news about Neko indeed. She really hoped Neko was still alive:
+
+> "For you I always stay alive... For without you I get lost in life. Wish you are still alive... "
+>
+> (Another version in Zhihu: "Only for you I stay alive... But you are not by my side. Isn't it? Wish you are still alive...")
+
+Yumao always missed Neko, and recalled the days with her.
+She often wrote down some caring words in her blog, such as prompted Neko to manage her hair, or told Neko to add or subtract clothes.
+
+In this period, Yumao tried to remember previous time and those things which is moving, happy and blissful.
+But her heart was took up by sadness, and her memory gradually worsen.
+She said that she forgot many things.
+
+She formerly planned to complete SRS within the year, but she didn't realize that because of her bad mental health and other reasons.
+
+In 30th June, all hope of her had been removed and she tried to take her own life.
+The next day, she left this world with regrets at her age of 17.
+
+> Wish that I could become a pretty girl who is kind-hearted, and become the sister of Ayaka and Ruri.
+
+In her blog post *[Separation and Reunion](https://web.archive.org/web/20210517104118/https://oao.moe/archives/948/)* published in march, she left many words to her friends, and thanked to their company all along. 
+
+<PhotoScroll photos={[
+    '${path}/photos/photo1.jpg',
+]} />
+
+## After Her Leaving
+
+In Zhihu question [What are your thoughts about ZhiHu user @yu-zhang-31-35 ?](https://www.zhihu.com/question/284818437), many friends wrote their memories of yumao, and expressed their sadness and regret of the death of her.
+They also felt regrettable for Neko's death due to close relationship between Neko and Yumao.
+
+In fact, Neko didn't pass away. Before her "death", in her email in 19th March, they still encouraged each other with the words "Must stay alive".
+
+Transgender community investigated this thing once, and found the fact that Neko was pronounced dead by her family.
```

Comment:
The transgender community investigated this, and found Neko was pronounced dead by her family.